translate/public/locale/en-US/translate.ftl

```diff
@@ -369,13 +369,13 @@ history-Translation--span-copied =
     .title = Copied ({ $machinerySources })
 
 history-translation--approved =
-    .title = Approved by { $user } on { DATETIME($reviewedDate, dateStyle:"long", timeStyle:"medium") }
+    .title = Approved by { $user } on { DATETIME($reviewedDate, month: "long", year: "numeric", day: "numeric", hour: "numeric", minute: "numeric", second: "numeric") }
```

Comment:
What's the purpose of this change?

Reply:
This change allows me to print the date without the Weekday as well. Is there a more succinct way of doing this?

Reply:
This format looks OK. It's the tooltip shown when you hover the translation creation date that is showing the weekday:
https://github.com/mozilla/pontoon/issues/2763#issuecomment-2203002727